### PR TITLE
Support for custom imports + tests for this scenario

### PR DESCRIPTION
**Pull request by kevin-smets**
Custom imports will be ignored. the train of thought is as follows:

File not found? -> assuming it's a custom import -> this entry will not be added to the graph.

### CODE REVIEW

File: sass-graph.js [hunk 1]
```diff
@@ -23,8 +23,11 @@ function resolveSassPath(sassPath, loadPaths) {
       return partialPath
     }
   }
-  var errMsg = "File to import not found or unreadable: " + sassPath;
-  throw errMsg;
+
+  var warnMsg = 'File to import not found or unreadable: assuming "' + scssPath +
```
Comment (xzyfer) [inline sass-graph.js]: For the moment just make this a comment and return `false`. 
I'll add a proper logging interface later.

File: sass-graph.js [hunk 2]
```diff
@@ -60,10 +63,10 @@ Graph.prototype.addFile = function(filepath, parent) {
       }
     }.bind(this));
     var resolved = resolveSassPath(imports[i], _.uniq(this.loadPaths));
-    if (!resolved) return false;
+    //if (!resolved) return false;
 
     // recurse into dependencies if not already enumerated
-    if(!_.contains(entry.imports, resolved)) {
+    if(resolved && !_.contains(entry.imports, resolved)) {
```
Comment (xzyfer) [inline sass-graph.js]: Doesn't look like this change is actually required. Please revert.

File: sass-graph.js [hunk 3]
```diff
@@ -60,10 +63,10 @@ Graph.prototype.addFile = function(filepath, parent) {
       }
     }.bind(this));
     var resolved = resolveSassPath(imports[i], _.uniq(this.loadPaths));
-    if (!resolved) return false;
+    //if (!resolved) return false;
```
Comment (xzyfer) [inline sass-graph.js]: Doesn't look like this change is actually required. Please revert.

Comment (kevin-smets) [inline sass-graph.js]: If I leave this in, not all imports for a file will be checked. Once an unresolved path has been encountered, it hits the return statement. All imports after an unresolved path will not be checked / added to the graph.

Comment (xzyfer) [inline sass-graph.js]: Ah I see, you're correct. Please change this to 

```
if (!resolved) continue;
```

**Comment by xzyfer** [comment]
Once you're done please squash your commits and I'll release this tomorrow as 1.1.0 :)

**Comment by kevin-smets** [comment]
Done and done :).

**Comment by kevin-smets** [comment]
Is this good to merge?

**Comment by xzyfer** [comment]
Hi @kevin-smets can you please rebase this of the current master?

**Comment by kevin-smets** [comment]
Done and done @xzyfer

**Comment by xzyfer** [comment]
Thanks @kevin-smets!

**Pull request merged**